crates/router/src/core/payments/transformers.rs

```diff
@@ -703,7 +703,12 @@ where
         .as_ref()
         .get_required_value("currency")?;
     let amount = currency
```

Comment:
Is this is total amount case or order_amount, why we are sending order amount, is this should be total_amount?

Reply:
This is order_amount only, we are sending order amount and currency in RedirectionFormData